src/python/Utils/procstats.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+#pylint: disable=
+"""
+File       : procstats.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: This module provide ability to get detailed information about
+certain UNIX process, referred by PID. It is based on psutil and threading
+python modules. It can be used in any web/data-server to get live status
+about the running server. Here is an example
+
+from Utils.procstats import status thread_stack
+
+
+class SomeServer(object):
+    def __init__(self):
+        self.time0 = time.time()
+    def status(self):
+	sdict = {'server': status()}
+        sdict['server'].update({'uptime': time.time()-self.time0})
+        sdict.update(thread_stack())
+	return sdict
+
+The exposed status endpoint will return a json dictionary containing server
+upteima and cpu/mem/threads information.
+"""
+
+# system modules
+import os
+import sys
+import json
+import time
+import argparse
+import threading
+import traceback
+
+try:
+    import psutil
+except:
+    pass
+
+def timestamp(time0=None):
+    "Standard timestamp"
+    ttime = time.localtime(time0) if time0 else time.localtime()
+    tstamp = time.strftime('%d/%b/%Y:%H:%M:%S', ttime)
```

Comment:
Not sure it makes too much difference, but one time format (suggested by the epoch converter page) that is clear, including the local time zone is:
`time.strftime("%a, %d %b %Y %H:%M:%S +0000", ttime)`
maybe we could update it?